doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multi-Orchestrator Example
+==========================
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+
+Below is a simple example demonstrating the process of:
+
```

Comment:
Maybe have the description of these steps directly below the list of the steps. I was a bit confused when the first heading was "Application script" when I was expecting an explanation od the launching of two standard Orchestrators, which I think you have lower down. Might be good just for easily finding the information?

Reply:
okay so I changed this, let me know if this its more clear, good find btw I can very much see where ur coming from